docs/user/README.md

```diff
@@ -0,0 +1,75 @@
+# Eventing Manager Module
+
+This module ships the Eventing Manager, which is a standard Kubernetes operator that observes the state of Eventing resources and reconciles them according to the desired state.
+
+With Kyma Eventing, you can focus on your business workflows and trigger them with events to implement asynchronous flows within Kyma. Generally, eventing consists of event producers (or publishers) and consumers (or subscribers) that send events to or receive events from an event processing backend.
+
+The objective of Eventing in Kyma is to simplify the process of publishing and subscribing to events. Kyma uses proven eventing backend technology to provide a seamless experience to the user with their end-to-end business flows. The user does not have to implement or integrate any intermediate backend or protocol.
+
+Kyma Eventing uses the following technology:
+
+- [NATS JetStream](https://docs.nats.io/) as backend within the cluster
```

Comment:
I guess this is only true nowadays when the user installs the NATS module?

Reply:
yes, but NATS is still the default backend whenever someone enable Eventing module.

Reply:
okay!